Neos.Neos/Documentation/References/NodeTypeDefinition.rst

```diff
@@ -144,19 +144,39 @@ The following options are allowed for defining a NodeType:
 
   ``icon``
     This setting defines the icon that the Neos UI will use to display the node type.
+    The icon can contain a custom SVG icon as a resource URI or a Fontawesome icon.
 
-    Legacy:
-    In Neos versions before 4.0 it was required to use icons from the Fontawesome 3 or 4 versions,
-    prefixed with "icon-"
+    Resource icon:
+    For custom SVG icons a resource URI to the file needs to be configured.
+    e.g. 'resource://Neos.Demo/Images/logo.svg'
 
-    Current:
-    In Neos 4.0, Fontawesome 5 was introduced, enabling the usage of all free Fontawesome icons:
+    Fontawesome icon:
+    All free Fontawesome 5 icons can be used:
     https://fontawesome.com/icons?d=gallery&m=free
-    Those can still be referenced via "icon-[name]", as the UI includes a fallback to the "fas"
+
+    Those can be referenced via "icon-[name]", as the UI includes a fallback to the "fas"
     prefix-classes. To be sure which icon will be used, they can also be referenced by their
     icon-classes, e.g. "fas fa-check".
 
 
+    ``previewIcon``
+    This setting defines the icon that will be used in the tile view of the NodeType selection dialog.
+    It is an additional icon that can be used to have more detailed icons for the node type. So that the users get a better idea of what the node type is for.
```

Comment:
```suggestion
    It is an additional icon that can be used to have more detailed icons for the node type. So that the users get a better idea of what the NodeType is for.
```